src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Extensions/IServiceCollectionExtensions.cs

```diff
@@ -89,7 +89,6 @@ public static void AddCompletionServices(this IServiceCollection services)
         services.AddSingleton<IRazorCompletionFactsService, LspRazorCompletionFactsService>();
         services.AddSingleton<IRazorCompletionItemProvider, DirectiveCompletionItemProvider>();
         services.AddSingleton<IRazorCompletionItemProvider, DirectiveAttributeCompletionItemProvider>();
-        services.AddSingleton<IRazorCompletionItemProvider, DirectiveAttributeParameterCompletionItemProvider>();
         services.AddSingleton<IRazorCompletionItemProvider, DirectiveAttributeEventParameterCompletionItemProvider>();
```

Comment:
Not for now of course, but the typename of this provider makes me wonder if we need to apply the same treatment to event attribute?

Reply:
It doesn't have near as much in common with the other directive attribute completion code, so I didn't consider trying to merge it with the others.